utils/create_fhir_store.sh

```diff
@@ -22,7 +22,34 @@ if [[ $# -ne 4 ]]; then
   exit 1
 fi
 
-# TODO create GCP Healthcare data-set too instead of assuming it exists.
+# Function that creates dataset and checks if the dataset already exist
```

Comment:
Consider documenting the fact that this expects what arguments, e.g,. like [this](https://google.github.io/styleguide/shellguide.html#function-comments).